### ISSUE TITLE
Consumers hang on offset out of range errors.

### ISSUE THREAD

**Issue by coughlanio** [tulios/kafkajs#334]
## Issue
Consumer appears to hang on an offset out of range error.

## Kafka Version
v2.1 via Confluent v5.1

## Code Snippets
```
    this.kafka = new Kafka({
      clientId: '001-raw-to-org',
      brokers: brokers.split(','),
      logLevel: logLevel.INFO,
    });

...

    this.consumer = this.kafka.consumer({
      groupId: '001-raw-to-org-group',
    });

...

    await this.consumer.subscribe({ topic: rawTopic, fromBeginning: true });

...

    await this.consumer.run({
      eachMessage: this.handleMessage.bind(this),
      autoCommitThreshold: 100,
    });
```
```
kafka-topics --zookeeper localhost:2181 --create --replication-factor 1 --partitions 100 --topic raw --config retention.ms=604800000 --config cleanup.policy=delete
```
## Error Logging
```
{"level":"ERROR","timestamp":"2019-04-09T09:51:41.477Z","logger":"kafkajs","message":"[ConsumerGroup] Offset out of range, resetting to default offset","topic":"raw","partition":75,"groupId":"001-raw-to-org-group","memberId":"001-raw-to-org-cbbd0a48-5e44-4fa7-9142-abc1f39528ef"}
```
## Logs
[Click here](https://gist.githubusercontent.com/coughlanio/743214b8bb4a7e04202bf7e32469a5d1/raw/2e31b3822d8a87ca7b00d1ec30b7bc4522455532/console.log)

**Comment by Nevon** [comment]
Is this a typo when you prepared the snippet, or is this actually what you are doing?

```
eachMessage: await this.handleMessage.bind(this)
```

**Comment by coughlanio** [comment]
Ah, sorry, the code is ```
    await this.consumer.run({
      eachMessage: this.handleMessage.bind(this),
      autoCommitThreshold: 100,
    });
```

**Comment by tulios** [comment]
Hi @coughlanio, I will take a look at this. We have been running the latest version of KafkaJS for a while in many teams/systems, and we haven't seen this happening. Is this a frequent error? Or a one-off thing?

**Comment by tulios** [comment]
I believe this issue was fixed in the latest versions, can you try *v1.13.0*?
I will close the issue for now, feel free to re-open if the issue is not fixed.
